### PR TITLE
Fetch gcal events only from custom Prox calendar

### PR DESCRIPTION
**Pull request by liuche**
Removes the 24/7 calendar, which simplifies the code.

### CODE REVIEW

**Comment by liuche** [comment]
[Add event url to Google Calendar description to be parsed into Event response to client.](https://trello.com/c/okO67hzw/247-add-event-url-to-google-calendar-description-to-be-parsed-into-event-response-to-client)

File: samples/fetch-gcal-test.py [hunk 1]
```diff
@@ -2,10 +2,6 @@
 import app.events as events
```
Comment (jhugman) [inline samples/fetch-gcal-test.py]: Suggest rename to `scripts/crawl_gcals`.

File: app/request_handler.py [hunk 2]
```diff
@@ -263,40 +263,23 @@ def loadCalendarEvents(timeDuration):
...
-                placeName = mapping['name']
-                location = mapping['location']
-                yelpId = _guessYelpId(placeName, location['lat'], location['lng'])
-                if yelpId:
-                    eventObj = representation.eventRecord(yelpId, location['lat'], location['lng'], summary, event['start']['dateTime'], event['end']['dateTime'], event['htmlLink'])
-                    return eventObj
-
-        except Exception as err:
-            print('Searching by name, error: {}'.format(err))
+            print('Searching for event location, error: {}'.format(err))
```
Comment (jhugman) [inline app/request_handler.py]:
```python
log.exception("getGcalEventObj")
```

**Review by jhugman — APPROVED** [review]
A few suggestions, but LGTM.

**Pull request merged**